### PR TITLE
PLATML-4296 Use default regression algorithm

### PR DESCRIPTION
**Pull request by mpaunovi**

### CODE REVIEW

**Review by tumulurik — APPROVED** [review]
LGTM, but pls wait for the merge until the pyspark-model-authoring-sdk is available from a public location which Wei is looking into.

**Review by sathyarun18 — APPROVED** [review]

**Comment by rjrussell77** [comment]
We are standardizing on running a linter prior to check-ins.  I quickly ran `pylint` and found a few issues to remedy.
```
pip install pylint
pylint myfile.py
```
Please do this for all files related to this check-in.

File: recipes/pyspark/pysparkretailapp/evaluator.py [hunk 1]
```diff
@@ -16,28 +16,16 @@
 #####################################################################
 
 from pyspark.sql.functions import col, avg, sqrt, abs, pow, lit
```
Comment (rjrussell77) [inline recipes/pyspark/pysparkretailapp/evaluator.py]: Not all of these imports are being used.

File: recipes/pyspark/pysparkretailapp/evaluator.py [hunk 2]
```diff
@@ -16,28 +16,16 @@
...
-    metric = [{"name": "MAPE", "value": mape, "valueType": "double"},
-              {"name": "MAE", "value": mae, "valueType": "double"},
-              {"name": "RMSE", "value": rmse, "valueType": "double"}]
-    return metric
\ No newline at end of file
+from sdk.evaluation.regression import RegressionEvaluator
+
+class Evaluator(RegressionEvaluator):
+    def __init__(self):
+       print ("Initiate")
```
Comment (rjrussell77) [inline recipes/pyspark/pysparkretailapp/evaluator.py]: should be, `print("Initiate")` (no space)

**Review by rjrussell77 — CHANGES_REQUESTED** [review]

File: recipes/pyspark/pysparkretailapp/evaluator.py (hunk 1 above)
Comment (mpaunovi) [reply recipes/pyspark/pysparkretailapp/evaluator.py]: Unused imports have been removed.

File: recipes/pyspark/pysparkretailapp/evaluator.py (hunk 2 above)
Comment (mpaunovi) [reply recipes/pyspark/pysparkretailapp/evaluator.py]: done

File: recipes/pyspark/setup.py [hunk 3]
```diff
@@ -24,5 +24,8 @@
       version='0.1.0',
       packages=['pysparkretailapp'],
       install_requires=required,
+      #we will use public Adobe github repo for sdk
+      dependency_links=[
+          'git+ssh://git@git.corp.adobe.com/ml/model-authoring-sdk-pyspark.git@master#egg=model_authoring_sdk'],
```
Comment (rjrussell77) [inline recipes/pyspark/setup.py]: I'm confused here - 'public Adobe github' and internal host (`corp.adobe.com`).

**Review by rjrussell77 — COMMENTED** [review]
Can you respond to the question related to the internal Adobe git ref?

File: recipes/pyspark/setup.py (hunk 3 above)
Comment (mpaunovi) [reply recipes/pyspark/setup.py]: Yes, this part is not really required. It was used during testing. The pyspark-model-authoring-sdk is now available from a public location. 

It's removed. Thanks.

**Pull request merged**